configs/recommended.js

```diff
@@ -10,7 +10,12 @@ module.exports = {
 		'unicorn/empty-brace-spaces': 'error',
 		'unicorn/error-message': 'error',
 		'unicorn/escape-case': 'error',
-		'unicorn/expiring-todo-comments': 'error',
+		'unicorn/expiring-todo-comments': [
+			"error",
+			{
+				"ignoreDatesOnPullRequests": true
```

Comment:
This is already the default: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/expiring-todo-comments.md#ignoredatesonpullrequests